R/digest.R

```diff
@@ -34,17 +34,6 @@ digest <- function(object, algo=c("md5", "sha1", "crc32", "sha256", "sha512",
     algo <- match.arg(algo)
     errormode <- match.arg(errormode)
 
-    .errorhandler <- function(txt, obj="", mode="stop") {
-        if (mode == "stop") {
-            stop(txt, obj, call.=FALSE)
-        } else if (mode == "warn") {	    # #nocov
-            warning(txt, obj, call.=FALSE)  # nocov
-            return(invisible(NA))           # nocov
-        } else {
-            return(invisible(NULL))         # nocov
-        }
-    }
-
```

Comment:
Moving this to another file is unrelated, and should really only happen after discussion.  

PRs are simply easier to deal with when they are _minimal_.

Reply:
Thanks for your feedback. Point well taken on this. I can remove this commit from the PR

Reply:
I would appreciate it.  I clearly see why you did it, and don't disagree on the merits.  But ... smaller diffs are simply better for PRs.  So if you could make that change I'd appreciate it.